src/components/ui/SuggestItem.jsx

```diff
@@ -1,61 +1,45 @@
 /* global _ */
 import React from 'react';
-import classNames from 'classnames';
 import NavigatorGeolocalisationPoi from 'src/adapters/poi/specials/navigator_geolocalisation_poi';
 import IconManager from '../../adapters/icon_manager';
 import Category from 'src/adapters/category';
 
 const ItemLabels = ({ firstLabel, secondLabel }) =>
-  <div className="autocomplete_suggestion__lines_container">
+  <div className="autocomplete_suggestion__labels">
     <div className="autocomplete_suggestion__first_line">{firstLabel}</div>
-    <div className="autocomplete_suggestion__second_line">{secondLabel}</div>
+    {secondLabel && <div className="autocomplete_suggestion__second_line">{secondLabel}</div>}
   </div>;
 
-const GeolocationItem = ({ isHighlighted }) =>
-  <div
-    className={classNames(
-      'autocomplete_suggestion itinerary_suggest_your_position',
-      { 'selected': isHighlighted }
-    )}
-    data-id="geolocalisation"
-    data-val={_('Your position', 'direction')}
-  >
-    <div className="itinerary_suggest_your_position_icon icon-pin_geoloc" />
-    {_('Your position', 'direction')}
+const GeolocationItem = () =>
+  <div className="autocomplete_suggestion autocomplete_suggestion--geoloc">
+    <div className="autocomplete-icon icon-pin_geoloc" />
+    <ItemLabels firstLabel={_('Your position', 'direction')} />
   </div>;
 
-const CategoryItem = ({ category, isHighlighted }) => {
+const CategoryItem = ({ category }) => {
   const { id, label, alternativeName, color, backgroundColor } = category;
   const icon = category.getIcon();
 
   return (
     <div
-      className={classNames(
-        'autocomplete_suggestion autocomplete_suggestion--category',
-        { 'selected': isHighlighted }
-      )}
+      className="autocomplete_suggestion autocomplete_suggestion--category"
       data-id={id}
-      data-val={label}
     >
       <div
         style={{ color, backgroundColor }}
-        className={`autocomplete-icon autocomplete-icon-rounded icon icon-${icon.iconClass}`}
+        className={`autocomplete-icon icon icon-${icon.iconClass}`}
       />
       <ItemLabels firstLabel={label} secondLabel={alternativeName} />
     </div>
   );
 };
 
-const PoiItem = ({ poi, isHighlighted }) => {
-  const { id, name, className, subClassName, type, alternativeName } = poi;
+const PoiItem = ({ poi }) => {
```

Comment:
I think we can even go further merging PoiItem and CategoryItem, they rally don't seem visally différent . This can be done in another PR I guess

Reply:
Maybe, but there are some differences in the icons, which are always a tricky part in the app because of the variety of icon fonts, sizes, etc.
As we'll have to introduce a new item type for NLU intentions, maybe further simplification will become obvious.